.github/workflows/pr_build.yml

```diff
@@ -59,10 +63,19 @@ jobs:
         echo "APP_TARGET_NAME=$APP_NAME" >> $GITHUB_ENV
 
         # Linux build variables
-        if [[ "${{ matrix.os }}" = "ubuntu-"* ]]; then
+        if [[ "${{ matrix.os }}" = "ubuntu-"* || "${{ matrix.os }}" = *"debian"* ]]; then
           echo "PYTHON_EXEC=python3" >> $GITHUB_ENV
           echo "INSTALLER_EXT=*" >> $GITHUB_ENV
-          echo "CMAKE_BUILD_EXTRA=-- -j3" >> $GITHUB_ENV
+          if [ "${{ matrix.os }}" = "self-hosted_debian-11_aarch64" ]; then
```

Comment:
Mm, this seems a bit overly specific maybe? It'll get outdated with new OS releases. Perhaps look in `/proc/cpuinfo` instead to get a CPU count?

Reply:
I will take a look at it.
GitHub also only provides two cores on the GitHub hosted Linux running as far as I remember, so this might make double sense.